README.md

```diff
@@ -56,7 +56,7 @@ _Long-form documentation for this section can be found [here](docs/build_and_loa
 ./phg annotate-fastas --keyfile /path/to/keyfile --output-dir /path/to/annotated/fastas --threads numberThreadstoRun
 
 ## Build VCF data
-./phg create-ranges --reference-file Ref.fa --gff my.gff --boundary gene --pad 500 -o /path/to/bed/file.bed
+./phg create-ranges --reference-file Ref.fa --gff my.gff --boundary gene --pad 500 --range-min-size 500 -o /path/to/bed/file.bed
```

Comment:
500 is the default?

Reply:
Yes - Originally Ed suggestion, but backed up by ShengKai's analysis of reelgene's minimum gene length for good genes.